src/features/code-input-and-control/code-input-and-control.tsx

```diff
@@ -0,0 +1,20 @@
+import CodeInputController from "../code-input-controller/code-input-controller";
+import CodeInput from "../code-input/code-input.module";
+
+interface CodeInputAndControllerProperties {
+  editor_id: number
+}
+
+export default function CodeInputAndController(props: CodeInputAndControllerProperties) {
+
+  return (
+    <div className="code-editor-and-controls">
+      <div className="code-editor-controls">
```

Comment:
Pack dieses div doch direkt in den `CodeInputController`, ohne wird die Komponente ja wahrscheinlich doof aussehen